static/src/stylesheets/module/site-messages/_fiv-banner.scss

```diff
@@ -23,45 +150,115 @@
     background-color: #ece4d9;
 }
 
+
 .fiv-banner__circles {
-    position: absolute;
     right: -80px;
     top: 50px;
+    isolation: isolate;
+    margin-right: 15px;
+
+
+    @include mq($from: tablet) {
+        position: relative;
+        right: auto;
+        top: auto;
+        margin-left: 25px;
+    }
 }
+
+$small-circle-height: 109px;
+$medium-circle-height: 138px;
+$large-circle-height: 214px;
+
 .fiv-banner__circle {
 
-    width: 187px;
-    height: 187px;
-    border-radius: 50%;
-    position: absolute;
-    right: 0;
+    width: $small-circle-height;
+    height: $small-circle-height;
+
+    @include mq($from: 375px) {
+        width: $medium-circle-height;
+        height: $medium-circle-height;
+    }
 
+    @include mq($from: wide) {
+        width: $large-circle-height;
+        height: $large-circle-height;
+    }
+    border-radius: 50%;
+    mix-blend-mode: screen;
```

Comment:
Ooooh, nice! How does this look on Edge / older browsers?

Reply:
still need to test on edge, but we had issues with safari and have a solution in this PR https://github.com/guardian/frontend/pull/21112/files#diff-95cd0213733bf7acf8ab941d34c6805cR184
This is all WIP currently